codespell_lib/data/dictionary.txt

```diff
@@ -4407,6 +4494,7 @@ arguemnt->argument
 arguemnts->arguments
 arguemtn->argument
 arguemtns->arguments
+arguent->argument
```

Comment:
Is it worth adding `argumentive` and `argumentation`/`argumentations` typos, perhaps even `argumentatively`?
```
arguentation->argumentation
arguentations->argumentations
arguentatively->argumentatively
arguentive->argumentive
```

Reply:
Not in this PR. These uncommon words need a lot of new entries.